packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -114,6 +115,134 @@ describe("Garbage Collection Tests", () => {
         sessionStorageConfigProvider.value.getRawConfig = oldRawConfig;
     });
 
+    describe("Configuration", () => {
+        const createGcWithPrivateMembers = (gcMetadata?: IGCMetadata, gcOptions?: IGCRuntimeOptions, snapshotCacheExpiryMs?: number) => {
+            const metadata: IContainerRuntimeMetadata | undefined = gcMetadata && { summaryFormatVersion: 1, message: undefined, ...gcMetadata };
+            const gcWithPrivates: IGarbageCollector & {
+                readonly gcEnabled: boolean;
+                readonly sweepEnabled: boolean;
+                readonly shouldRunGC: boolean;
```

Comment:
I've noticed `shouldRunGC` isn't being tested, this is the more important variable for mark phase.